coc/war_clans.py

```diff
@@ -147,6 +147,14 @@ def defenses(self) -> typing.List["WarAttack"]:
         """
         other = self._war and self._war.clan if self.is_opponent else self._war.opponent
         return other.attacks if other else []
+    
+    @property
+    def average_attack_duration(self) -> int:
+        """:class:`int`: Returns the average duration of all clan member's attacks this war."""
+        total_duration = 0
```

Comment:
this line probably doesn't need to be here, since you just re-declare it below no matter what

Reply:
removed